go/v1beta1/storage/filtering/types.go

```diff
@@ -28,7 +35,8 @@ type Term map[string]string
 type Operation string
 
 const (
-	AndOperation   Operation = "_&&_"
-	OrOperation    Operation = "_||_"
-	EqualOperation Operation = "_==_"
+	AndOperation      Operation = "_&&_"
+	OrOperation       Operation = "_||_"
+	EqualOperation    Operation = "_==_"
+	NotEqualOperation Operation = "_!=_"
```

Comment:
Do we need to redefine what cel does? https://godoc.org/github.com/google/cel-go/common/operators